executor/tiflash_test.go

```diff
@@ -138,4 +137,9 @@ func (s *tiflashTestSuite) TestMppExecution(c *C) {
 	tk.MustQuery("select avg(t1.a) from t1 , t where t1.a = t.a").Check(testkit.Rows("2.0000"))
 	// test proj and selection
 	tk.MustQuery("select count(*) from (select a * 2 as a from t1) t1 , (select b + 4 as a from t)t where t1.a = t.a").Check(testkit.Rows("3"))
+
+	// test shuffle hash join.
+	tk.MustExec("set @@session.tidb_broadcast_join_threshold_size=1")
+	tk.MustQuery("select count(*) from t1 , t where t1.a = t.a").Check(testkit.Rows("3"))
+	tk.MustQuery("select count(*) from t1 , t, t2 where t1.a = t.a and t2.a = t.a").Check(testkit.Rows("3"))
```

Comment:
... obtained string = "[tikv:1]unsupported executor type TypeSelection"
... expected string = ""
... sql:select b from t3 where a >1, 

can this SQL run correctly? which errors before several days.

Reply:
Of course